internal/plugin/grpc_client_plugin.go

```diff
@@ -124,10 +126,24 @@ func (gc *GrpcClient) Info() *bus.Info {
 	}
 }
 
-func (gc *GrpcClient) Process(_ context.Context, _ *bus.Message) {}
+func (gc *GrpcClient) Process(ctx context.Context, msg *bus.Message) {
+	switch {
+	case msg.Topic == bus.InstancesTopic:
+		if newInstances, ok := msg.Data.([]*v1.Instance); ok {
+			err := gc.sendDataPlaneStatusUpdate(ctx, newInstances)
+			if err != nil {
+				slog.Error("Unable to send data plane status update", "error", err)
```

Comment:
Is there no `slog.ErrorContext`?
```suggestion
				slog.ErrorContext(ctx, "Unable to send data plane status update", "error", err)
```